autogpt_platform/backend/.env.example

```diff
@@ -9,13 +9,24 @@ REDIS_HOST=localhost
 REDIS_PORT=6379
 REDIS_PASSWORD=password
 
-ENABLE_AUTH=false
 ENABLE_CREDIT=false
 APP_ENV="local"
 PYRO_HOST=localhost
 SENTRY_DSN=
-# This is needed when ENABLE_AUTH is true
-SUPABASE_JWT_SECRET=our-super-secret-jwt-token-with-at-least-32-characters-long
+
+## User auth with Supabase is required for any of the 3rd party integrations with auth to work.
```

Comment:
Add the FRONTEND_BASE_URL here or it won't redirect correctly in my testing

Reply:
done